backend/cmm.ml

```diff
@@ -218,11 +218,9 @@ and operation =
   | Copaque
   | Cbeginregion | Cendregion
 
-type value_kind =
-  | Vval of Lambda.value_kind (* Valid OCaml values *)
-  | Vint (* Untagged integers and off-heap pointers *)
-  | Vaddr (* Derived pointers *)
-  | Vfloat (* Unboxed floating-point numbers *)
+type kind_for_unboxing =
```

Comment:
Can you add a small comment stating that this is used for unboxing specifically during the `cmmgen` pass ? (and thus not relevant for flambda2)